### PR TITLE
Carousel added for photography page

### PR DESCRIPTION
**Pull request by cjinc121**

### CODE REVIEW

File: src/pages/PhotographyCarousel.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import { BsFillCaretLeftFill, BsFillCaretRightFill } from "react-icons/bs";
+import { useState } from "react";
+import { AiOutlineClose } from "react-icons/ai";
+
+const PhotographyCarousel = () => {
+  const { topicNo } = useParams();
+  const [imgNo, setImgNo] = useState(topicNo[2]);
+  const navigate = useNavigate();
+
+  const imgSrcArray = photographyPageContent.filter(
```
Comment (ayush20892) [inline src/pages/PhotographyCarousel.js:13]: Give proper Spacing

File: src/pages/PhotographyCarousel.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      </div>
+
+      <BsFillCaretRightFill
+        onClick={rightNavigation}
+        className="nav-arrow nav-arrow-right"
+      />
+    </div>
+  );
+};
+
```
Comment (ayush20892) [inline src/pages/PhotographyCarousel.js:59]: Nice work

File: src/assets/css/pages/PhotographyCarousel.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+.main-img {
+  position: relative;
+  margin: 0px;
+}
+.img-no {
+  color: white;
+  font-weight: bold;
+  position: absolute;
+  bottom: 20px;
+  left: 50%;
```
Comment (ayush20892) [inline src/assets/css/pages/PhotographyCarousel.css:29]: Add transform: translateX(-50%);

**Review by ayush20892 — CHANGES_REQUESTED** [review]

File: src/assets/css/pages/PhotographyCarousel.css (hunk 3 above)
Comment (cjinc121) [reply src/assets/css/pages/PhotographyCarousel.css:29]: resolved

**Review by ayush20892 — APPROVED** [review]

**Pull request merged**